webrtc-stats.html

```diff
@@ -1428,7 +1428,9 @@
               </dt>
               <dd>
                 <p>
-                  Only valid for video. It represents the nominal FPS value.
+                  Only valid for video. It represents the nominal FPS value before the degradation
+                  preference is applied. For sending tracks it is the current captured FPS
+                  and for the receiving tracks it is the current decoding framerate.
```

Comment:
How is the FPS calculated? Over the last second, based on the time between the last two frames, etc?